jsk_perception/src/robot_to_mask_image.cpp

```diff
@@ -66,10 +66,19 @@ namespace jsk_perception
       image_geometry::PinholeCameraModel model;
       model.fromCameraInfo(info_msg);
 
-      cv::Mat mask_image = cv::Mat::zeros(info_msg->height, info_msg->width, CV_8UC1);
+      int height = info_msg->height;
```

Comment:
This PR is awesome because of considering `sensor_msgs/CameraInfo`'s `binning`.
Considering also `roi`, this PR is more than perfect.

http://docs.ros.org/en/melodic/api/sensor_msgs/html/msg/CameraInfo.html

> Region of interest (subwindow of full camera resolution), given in full resolution (unbinned) image coordinates.


The following code may be helpful in calculating internal parameters.
https://github.com/iory/cameramodels/blob/c9831271f385db7d6da515b87034491ebfb50b00/cameramodels/pinhole_camera.py#L185-L192

Reply:
Thank you! Updated.
You can use these params in `image_proc/crop_decimate` to add roi to camera info 
```
 <param name="x_offset" type="int" value="200" />
 <param name="y_offset" type="int" value="100" />
 <param name="width" type="int" value="200" />
 <param name="height" type="int" value="320" />
```

Reply:
Thats' good.
I send a PR to check the results.
https://github.com/nakane11/jsk_recognition/pull/2
Please check it.

Reply:
Thank you. I checked it.